packages/packages/api/src/devices/client/metriport.ts

```diff
@@ -98,6 +98,18 @@ export class MetriportDevicesApi {
     });
   }
 
+  /**
+   * For the given user ID, revoke the user's access to the specified provider.
+   *
+   * @param {string} userId    - The user ID of the user for which to revoke access.
+   * @returns void.
+   */
+  async deleteUser(userId: string): Promise<void> {
+    await this.api.delete("/user/delete", {
+      params: { userId: userId },
```

Comment:
Defs NABD and a nitpick: can use `{ userId }` when both the prop and the const/variable have the same name.